2015-Fall/nativix/viewControler.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  ViewController.swift
+//  Calculator
+//
+//  Created by MARIA NATIVIDAD LARA DIAZ on 01/09/15.
+//  Copyright (c) 2015 Appstract Development. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var display: UILabel!
+    
+    var userIsInTheMiddleOfTypingANumber: Bool = false
```

Comment:
The variable type is clear from the context. So you could just write `var userIsInTheMiddleOfTypingANumber = false`